pkg/helmclient/installer.go

```diff
@@ -0,0 +1,91 @@
+package helmclient
+
+import (
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+// InstallResult represents the results for an install action.
+// It contains the underlying Release
+type InstallResult struct {
+	helmrelease *release.Release
+	Release     *Release
+	Data        string
```

Comment:
we should get json data, and inturn write it as object and not direct string dump into a field. If we gonna allow yaml we could format it and then write it (not critical)